### PR TITLE
face mask detection using computer vision added

### PR DESCRIPTION
**Pull request by 17arindam**
# Related Issues or bug
I have added Face Mask Detection System built with OpenCV, Keras/TensorFlow using Deep Learning and Computer Vision concepts in order to detect face masks in static images as well as in real-time video streams.

Fixes: #160 

# Proposed Changes
added a face detection model

  
# Screenshots
image

### CODE REVIEW

**Comment by 17arindam** [comment]
@UppuluriKalyani please review my pr

**Comment by 17arindam** [comment]
@SaiNivedh26 please assign me level for this pr
